### ISSUE TITLE
jira init ignores JIRA_AUTH_TYPE and prompts instead

### ISSUE THREAD

**Issue by biskinis** [ankitpokhrel/jira-cli#704]
**Describe the bug**
In version 1.5, `jira init` appears to ignore `JIRA_AUTH_TYPE` and always prompts for the authentication type.

1. JiraCLI Version:

```

(Version="1.5.0", GitCommit="8449ab0c27b60caf7ca8bc5fdd5fff876e3672f5", CommitDate="2024-01-09T20:30:43+00:00", GoVersion="go1.21.5", Compiler="gc", Platform="linux/amd64")

```

2. Are you using Jira cloud or on-premise jira server?

```
SERVER INFO
-----------

Version:         9.5.1
Build Number:    950001
Deployment Type: Server
Default Locale:
```

4. What operating system are you using? Also mention version.

```
PRETTY_NAME="Debian GNU/Linux 11 (bullseye)"
NAME="Debian GNU/Linux"
VERSION_ID="11"
VERSION="11 (bullseye)"
VERSION_CODENAME=bullseye
ID=debian
HOME_URL="https://www.debian.org/"
SUPPORT_URL="https://www.debian.org/support"
BUG_REPORT_URL="https://bugs.debian.org/"
```

5. What terminal are you using? Also mention version.

```
Running in Docker container.
```

**To Reproduce**

```
$ export JIRA_API_TOKEN='secret'
$ export JIRA_AUTH_TYPE='bearer'
$ jira init --installation=local --login my.username --board 'My Board' --project MYPROJ --server https://my.jira --force
```

**Expected behavior**
Expect the config file to be generated immediately with no console interaction.  Instead I am prompted for the authentication type.  Once that is selected the file is generated.

**Additional context**
This seems to be due to changes between 1.4 and 1.5.  Reverting to 1.4 restores the original behaviour.

**Comment by ankitpokhrel** [comment]
Hi @biskinis, this should already be fixed with https://github.com/ankitpokhrel/jira-cli/pull/701. 

I will make a new release this weekend. Meanwhile if you wish to test the fix,  you can do so by installing the main branch.

```
# Docker
$ docker run -it --rm ghcr.io/ankitpokhrel/jira-cli:main

# Golang
$ go install github.com/ankitpokhrel/jira-cli/cmd/jira@main

# Homebrew
$ brew tap ankitpokhrel/jira-cli
$ brew install jira-cli --HEAD
```

**Comment by ankitpokhrel** [comment]
This should be fixed in the latest release [v1.5.1](https://github.com/ankitpokhrel/jira-cli/releases/tag/v1.5.1)
